source/lex.tex

```diff
@@ -208,10 +208,10 @@
 \end{bnf}
 
 The character designated by the \grammarterm{universal-character-name} \tcode{\textbackslash
-UNNNNNNNN} is that character whose character short name in ISO/IEC 10646 is
-\tcode{NNNNNNNN}; the character designated by the \grammarterm{universal-character-name}
-\tcode{\textbackslash uNNNN} is that character whose character short name in
-ISO/IEC 10646 is \tcode{0000NNNN}. If the hexadecimal value for a
+U00NNNNNN} is that character whose character short identifier in ISO/IEC 10646 is
+\tcode{NNNNNN}; the character designated by the \grammarterm{universal-character-name}
+\tcode{\textbackslash uNNNN} is that character whose character short identifier in
+ISO/IEC 10646 is \tcode{NNNN}. If the hexadecimal value for a
```

Comment:
Sorry for the driveby, but why do we say "hexadecimal value"? Why not just "value"? In which way does the value depend on a particular serialization format?

Reply:
Can we keep that separate, please? This is enough of a tar pit already, and might benefit from a more wholesale rework.